examples/README.md

```diff
@@ -0,0 +1,13 @@
+# Examples
+
+This is a collection of examples that show how you could approach some use cases in fetching and
```

Comment:
What about `[...] that show how to interact with Astarte in common user cases.`

Reply:
That's better, I agree. Fixed it.